package.json

```diff
@@ -21,20 +21,10 @@
     "material components",
     "material design"
   ],
-  "contributors": [
```

Comment:
I think so:
![prateekbh][https://github.com/prateekbh.png?size=200] ![filoozom][https://github.com/filoozom.png?size=200]

So you do prefer a ```CONTRIBUTORS.md``` file instead of just linking to the contributors page on GitHub?

Reply:
yes, we'll add a link to it on readme. sorting can be just alphabetical